src/main/java/com/eva/model/person/applicant/application/Experience.java

```diff
@@ -0,0 +1,102 @@
+package com.eva.model.person.applicant.application;
+
+import java.time.LocalDate;
+import java.time.DateTimeException;
+
+/**
+ * Represents an applicant's experience at a particular company, similar to what is seen in a resume.
+ */
+public class Experience {
+    private LocalDate startDate;
+    private LocalDate endDate;
+    private String companyName;
+    private String position;
+    private String description;
+
+    /**
+     * Creates an experience object.
+     * @param startDate The start date.
+     * @param endDate The end date.
+     * @param companyName The name of the company.
+     * @param position The position held.
+     * @param description The description of the job.
+     */
+    public Experience(String startDate, String endDate, String companyName, String position, String description) {
+        this.startDate = parseDate(startDate);
+        this.endDate = parseDate(endDate);
+        this.companyName = companyName;
+        this.position = position;
+        this.description = description;
+    }
+
+    private static LocalDate parseDate(String date) throws DateTimeException {
+        int day = Integer.parseInt(date.substring(0, 2));
+        int month = Integer.parseInt(date.substring(2, 4));
+        int year = Integer.parseInt(date.substring(4, 8));
+        return LocalDate.of(year, month, day);
+    }
```

Comment:
perhaps we can abstract this out to dateutil? If the current method doesnt suit what you need you can create another method and another datetimeformatter to suit what you need

Reply:
yup ok